decktape.js

```diff
@@ -2,22 +2,22 @@
 
 'use strict';
 
-const chalk     = require('chalk'),
-      crypto    = require('crypto'),
-      Font      = require('fonteditor-core').Font,
-      fs        = require('fs'),
-      os        = require('os'),
-      parser    = require('./libs/nomnom'),
-      path      = require('path'),
-      puppeteer = require('puppeteer'),
-      URI       = require('urijs'),
-      util      = require('util');
-
-const { PDFDocument, PDFName, ParseSpeeds, decodePDFRawStream } = require('pdf-lib');
-
-const { delay, pause } = require('./libs/util');
-
-const plugins = loadAvailablePlugins(path.join(path.dirname(__filename), 'plugins'));
+import chalk             from 'chalk';
+import chalkTemplate     from 'chalk-template';
```

Comment:
Chalk [v5](https://github.com/chalk/chalk/releases/tag/v5.0.0) seems to have split out its tagged template literal functionality into a separate package 🤷‍♂️